test/test.ts

```diff
@@ -862,59 +862,56 @@ function runTests(targetPlatform: platform.IPlatform, useWkWebView: boolean): vo
             }
         });
 
-        describe("#localPackage.installOnNextRestart2x", function() {
+        if (!onlyRunCoreTests) {
```

Comment:
interesting...i'm changing the whole implementation pretty soon (with the change that will support react-native testing) but this is good to know